benchmarks/templates/benchmarks/components/left-sidebar.html

```diff
@@ -0,0 +1,4 @@
+<a class="selected" href="#">Leaderboard</a>
+<a href="#">Compare</a>
+<a href="#">Tutorials</a>
+<a href="#">Profile</a>
```

Comment:
hrefs missing?

Reply:
these are just placeholders so that there was something present in the layout. I'll add the routing and hrefs later